// @ceiba/toolbar/fesm2015/ceiba-toolbar.js
import { Component, NgModule } from '@angular/core';

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
class ToolbarComponent {
    constructor() { }
    /**
     * @return {?}
     */
    ngOnInit() {
    }
}
ToolbarComponent.decorators = [
    { type: Component, args: [{
                selector: 'app-toolbar',
                template: "<h1>Angular Seed</h1>\r\n<div class=\"more\"></div>\r\n",
                styles: [":host{background-color:#106cc8;color:rgba(255,255,255,.87);display:block;height:48px;padding:0 16px}h1{display:inline;font-size:20px;font-weight:400;letter-spacing:.1px;line-height:48px}.more{background:url(/assets/svg/more.svg);float:right;height:24px;margin-top:12px;width:24px}"]
            }] }
];
/** @nocollapse */
ToolbarComponent.ctorParameters = () => [];

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
class ToolbarModule {
}
ToolbarModule.decorators = [
    { type: NgModule, args: [{
                declarations: [ToolbarComponent],
                imports: [],
                exports: [ToolbarComponent]
            },] }
];

export { ToolbarComponent, ToolbarModule };
